tests/basic/storage_pool_test.go

```diff
@@ -6302,20 +6328,787 @@ var _ = Describe("{PoolResizeInvalidPoolID}", func() {
+			fmt.Sprintf("Verify pool %s on expansion using auto option", poolUUID))
+
+	})
+
+	JustAfterEach(func() {
+		defer EndTorpedoTest()
+		log.InfoD("Exit from Maintenance mode if Pool is still in Maintenance")
+		log.FailOnError(ExitNodesFromMaintenanceMode(), "exit from maintenance mode failed?")
+		AfterEachTest(contexts, testrailID, runID)
+	})
+
+})
+
+var _ = Describe("{AddMultipleDriveStorageLessNodeResizeDisk}", func() {
+	/*
+		Pool Resize after adding drives to storage less node
+		https://portworx.testrail.net/index.php?/cases/view/51329
+		https://portworx.testrail.net/index.php?/cases/view/51330
+	*/
+
+	var testrailID = 0
+	// Testrail Description : Pool Resize after adding drives to storage less node
+	var runID int
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("AddMultipleDriveStorageLessNodeResizeDisk",
+			"Add Drive to storage less node and resize Disk",
+			nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+	var contexts []*scheduler.Context
+	stepLog := "Add Drives to storage less node and resize after adding the node"
+	It(stepLog, func() {
+		log.InfoD(stepLog)
+
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("storagelessresizedisk-%d", i))...)
+		}
+		ValidateApplications(contexts)
+		defer appsValidateAndDestroy(contexts)
+
+		// Get Pool with running IO on the cluster
+		poolUUID, err := GetPoolIDWithIOs(contexts)
+		log.FailOnError(err, "Failed to get pool running with IO")
+		log.InfoD("Pool UUID on which IO is running [%s]", poolUUID)
+
+		// Get Node Details of the Pool with IO
+		nodeDetail, err := GetNodeWithGivenPoolID(poolUUID)
+		log.FailOnError(err, "Failed to get Node Details from PoolUUID [%v]", poolUUID)
+		log.InfoD("Pool with UUID [%v] present in Node [%v]", poolUUID, nodeDetail.Name)
+
+		storageLessNode := node.GetStorageLessNodes()
+		// Get random storage less node present in the cluster
+		var pickNode node.Node
+		if len(storageLessNode) == 0 {
+			if IsEksPxOperator() != true {
+				log.FailOnError(fmt.Errorf("DeletePool is currently supported for EKS and LocalDrives"), "Pool deletion supported?")
+			}
+			err := MakeStoragetoStoragelessNode(*nodeDetail)
+			log.FailOnError(err, "failed to mark storage Node to Storage less Node")
+			storageLessNode = node.GetStorageLessNodes()
+		}
+		randomIndex := rand.Intn(len(storageLessNode))
+		pickNode = storageLessNode[randomIndex]
+		log.InfoD("Storage Less node is [%v]", pickNode.Name)
+
+		// Add multiple Drives to Storage less node
+		maxDrivesToAdd := 6
+		for i := 0; i < maxDrivesToAdd; i++ {
+			log.InfoD("Adding [%d/%d] disks to the Node [%v]", i, maxDrivesToAdd, pickNode.Name)
+			log.FailOnError(addCloudDrive(pickNode, -1), "error adding cloud drive on Node [%v]", pickNode.Name)
+		}
+
+		// Resize the cloud drive added on the Node
+		poolList, err := GetPoolsDetailsOnNode(pickNode)
```

Comment:
add err = Inst().V.RefreshDriverEndpoints() before this so that node will populate with new pool details

Reply:
done